man/systemd.unit.xml

```diff
@@ -1031,6 +1033,17 @@
         whether a certain path exists, is a regular file and marked
         executable.</para>
 
+        <para><varname>ConditionUser=</varname> takes a numeric
+        <literal>UID</literal> or a UNIX user name. This condition
+        may be used to check whether the service manager is running
+        as the given user. This option is only meaningful for the user
+        managers.</para>
```

Comment:
probably should say that we check both UID and EUID, even though it doesn't matter IRL...

Also, I'd not use the (strong) wording "meaningful" here, but rather say something like: "this option is not useful for system services, as the system manager exclusively runs as "root" user, and the test result is hence constant". or something like that...